.changes/bundler-gnueabi-armel.md

```diff
@@ -0,0 +1,5 @@
+---
+"tauri-bundler": major:bug
```

Comment:
Thank you the review @lucasfernog!

I had chosen `major` instead of `patch` because this PR changes an API (the signature of `tauri_bundler::bundle::Settings::binary_arch`). Is semver only for public Tauri API changes?